runtime.ts

```diff
@@ -1142,16 +1142,27 @@ module J2ME {
     return null;
   }
 
+  function reportError(method, key) {
+    return function() {
+      try {
+        return method.apply(this, arguments);
+      } catch (e) {
+        stderrWriter.errorLn("Native " + key + " throws:" + e);
```

Comment:
The Java exceptions are not unexpected error from the native method. They should be handled properly by the outer java functions.

So I fix this by not reporting the Java exceptions: 3b2991ce093.